lib/src/main/java/com/diffplug/spotless/PaddedCell.java

```diff
@@ -150,25 +150,20 @@ public boolean isResolvable() {
 
 	/** Returns the "canonical" form for this particular result (only possible if isResolvable). */
 	public String canonical() {
-		// @formatter:off
```

Comment:
i know it looks intended and i know you mentioned it recently, having done this (IDE) suggedtion already.

Can we evlevate this or do we need this form explizicly, then we could suppress these rare cases. Some of this pattern makes sense on some of the parts done.